message_adapter/message_adapter.py

```diff
@@ -54,18 +53,17 @@ def load_and_update_remote_event(self, incoming_event, context):
         else:
             event = load_remote_event(event)
 
-        if context and 'meta' in event and 'workflow_tasks' in event['meta']:
-            cumulus_meta = event['cumulus_meta']
+        if context and 'meta' in event:
```

Comment:
Slight logic change here -- we're adding this regardless of workflow tasks being defined.    This both makes units better *and* seems like the usual use case given the workflow message template.  

In cases where users aren't using this with Cumulus, this just results in slight default bloat of the meta object.      I'm going to note it in the CHANGELOG, but am open to pushback.